app/controllers/friendship_request_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class FriendshipsController < ApplicationController
```

Comment:
Why this class has the same name as the class in the `friendships_controller.rb` file? If you don't need this file you can remove it.

Reply:
the `friendship_controller.rb` file was created by mistake when we were trying to create the controllers from the command line, actually It can be deleted without any consequence, I did It and all the testing passes, the Functional controller for friendship model is the `FriendshipsController`